SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -185,6 +188,8 @@
 
   private long startNewReconcileEverySeconds = TimeUnit.MINUTES.toSeconds(10);
 
+  private TimeZone timeZone = TimeZone.getTimeZone("UTC");
```

Comment:
Will this initialize correctly from a String? (i.e. reading it from a yaml field set to 'EST' or 'UTC') It very well might, but something we should probably test to be sure